rosetta/rosetta/projects/pax/README.md

```diff
@@ -22,16 +22,17 @@ where `DATASET_PATH` is the path to the Pile or Lambada dataset. If these datase
 The given models are trained using The Pile dataset and evaluated using the Lambada dataset. The scripts [download_the_pile.py](https://github.com/google/paxml/blob/main/paxml/contrib/gpu/scripts_gpu/download_the_pile.py) and [download_lambada.py](https://github.com/google/paxml/blob/main/paxml/contrib/gpu/scripts_gpu/download_lambada.py) will download The Pile and the Lambada datasets to the `TFDS_DATA_DIR` enviroment variable. To control the location of the downloaded datasets, use the following command prior to running the download scripts: `export TFDS_DATA_DIR=<path_to_dowload_data_to>`. After the data has been successfully downloaded, use the same `TFDS_DATA_DIR` when running experiments.
 
 ## Downloading the SentencePiece Model
-Pax models require a pretrained SentencePiece model to tokenize the datasets. The SentencePiece model used in the following experiments is `gs://mlperf-llm-public2/vocab/c4_en_301_5Mexp2_spm.model`. This model was trained using [these instructions](https://github.com/sgpyc/training/blob/paxml-llm-draft/large_language_model/paxml/utils/generate_spm.md). See below for information on downloading `gs://mlperf-llm-public2/vocab/c4_en_301_5Mexp2_spm.model` from Google Cloud. 
-
-1. Ensure you have the [Google Clould SDK](https://cloud.google.com/sdk/docs/install) installed.
-2. Log into the Cloud by using the following command: `gcloud auth login` and following the prompts. 
-3. Once logged in, use the following command to download the vocab file to your current working directory: 
+Pax models require a pretrained SentencePiece model to tokenize the datasets. The SentencePiece model used in the following experiments is `gs://mlperf-llm-public2/vocab/c4_en_301_5Mexp2_spm.model`. This model was trained using [these instructions](https://github.com/sgpyc/training/blob/paxml-llm-draft/large_language_model/paxml/utils/generate_spm.md). Use the following commands to download the tokenizer locally. This should be done _prior_ to launching the container.  
 ```
-gsutil -m cp -r gs://mlperf-llm-public2/vocab/c4_en_301_5Mexp2_spm.model .
+mkdir c4_sentencepiece
+cd c4_sentencepiece
+wget https://github.com/nvjax-svc-0/assets/blob/main/sentencepiece_c4/c4_en_301_5Mexp2_spm.model
+cd ..
```

Comment:
This won't quite work, you need the raw url, not the blob. You can also avoid the double `cd`s by just specifying the dir
```suggestion
wget -P c4_sentencepiece https://github.com/nvjax-svc-0/assets/raw/main/sentencepiece_c4/c4_en_301_5Mexp2_spm.model
```